### PR TITLE
Update README.txt

### PR DESCRIPTION
**Pull request by HeshamIsmail**
## :memo: Description

Our copywriting team has created an introductory page for the WooCommerce Dropshipping Plugin page that should be added as soon as possible.

You may find the page in the link below:

https://docs.google.com/document/d/1Q5MhImAStKRYqgeGprAUSabCiLT0IdmBTkWkWNrZzXw/edit### :dart: Relevant issues

### :gem: Type of change

- Plug-in Description modifications

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -7,43 +7,57 @@ Tested up to: 5.6.0
 Stable tag: 2.2.0
 WC tested up to: 4.8.0
 
-== Description ==
+=== Description Page ===
```
Comment (HazemKhaled) [inline README.txt:10]: required by wordpress.org to be == not. ===

File: README.txt [hunk 2]
```diff
@@ -7,43 +7,57 @@ Tested up to: 5.6.0
...
 
-===Knawat WooCommerce DropShipping:===
+Knawat - WooCommerce Dropshipping
```
Comment (HazemKhaled) [inline README.txt:12]: why not title?
===Knawat - WooCommerce Dropshipping===

File: README.txt [hunk 3]
```diff
@@ -7,43 +7,57 @@ Tested up to: 5.6.0
...
-==Features==
+Launching your e-business or online store has never been as easy. All you need to do is choose the type, the number of your products, and the desired language, click on it, and there it will be; all the details of these products, such as; images, description, and data will be in your store.
 
-We ship to every corner around the globe. Sell your products to customers around the world, Knawat will pack and ship orders directly to your customer doorstep with your invoice and your brand name. Automatic updating prices and stock level. Focus your efforts on selling not on stock management. Stock levels and product prices are automatically updated in real-time with the suppliers’ system.
+The products will be imported to your products catalog at the wholesale price. Price and stock will be both updated in real-time whenever they are modified either by the suppliers or by modifying your profit margin.
 
-===Multilingual Content===
 
-Our translation team working on product description translating it to three different languages (English, Arabic, and Turkish), and localized to different cultures. Reach your target customers with the right content.
+=== Features ===
```
Comment (HazemKhaled) [inline README.txt:29]: keep it == instead of ===

**Review by HazemKhaled — CHANGES_REQUESTED** [review]

File: README.txt (hunk 1 above)
Comment (HeshamIsmail) [reply README.txt]: Updated upon your request

File: README.txt (hunk 2 above)
Comment (HeshamIsmail) [reply README.txt]: Updated upon your request

File: README.txt (hunk 3 above)
Comment (HeshamIsmail) [reply README.txt]: Updated upon your request

**Pull request merged**